<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="L76307" failures="1" tests="3" name="Test" time="26.454" errors="0" timestamp="2024-10-21T15:57:49 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.274"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.287">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.745"/>
</testsuite> <!-- Test -->
